src/themes/partials/variables/_light-variables.scss

```diff
@@ -0,0 +1,7 @@
+$dark: #333;
+$interaction: #add8e6;
```

Comment:
This either has to be `interaction-color` or all dependencies in the light theme stylesheet have to be changed to `interaction`. It throws an error when you use the light theme.

Please check it thoroughly for both themes.